### PR TITLE
#163421750 Added logic to enable users to comment

### PR DESCRIPTION
**Pull request by oldmonad**

### CODE REVIEW

File: server/controller/comments.js [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
+    const checkquestion = await QuestionModel.getQuestionById(newComment.questionId);
 
-      const newComment = new CommentsModel(comment);
-      const createdComment = await CommentsModel.createComment(newComment);
+    if (!checkquestion) {
+      return errorResponse(res, 404, 'Question does not exist');
     }
+
+    newComment.title = checkquestion.title;
+    newComment.body = checkquestion.body;
```
Comment (Veraclins) [inline server/controller/comments.js]: Why are you duplicating question title and body on the comment? I think the only thing the comment needs from the question is the Id.

File: server/routes/meetups.js [hunk 2]
```diff
@@ -12,12 +12,13 @@ import rsvpController from '../controller/rsvp';
...
 
 router.post('/', Auth.verifyToken, Auth.adminAuth, meetupValidation.validCreateMeetup, meetupValidation.checkDate, tryCatch(meetupController.createMeetup));
 router.get('/upcoming/', Auth.verifyToken, tryCatch(meetupController.getUpcomingMeetups));
-router.get('/:id', Auth.verifyToken, tryCatch(meetupController.getSingleMeetup));
+router.get('/:id', Auth.verifyToken, validateId, tryCatch(meetupController.getSingleMeetup));
 router.get('/', Auth.verifyToken, tryCatch(meetupController.getAllMeetups));
-router.patch('/:meetupId/rsvps', Auth.verifyToken, tryCatch(rsvpController.respondToRsvp));
-router.delete('/:id', Auth.verifyToken, Auth.adminAuth, tryCatch(meetupController.deleteMeetup));
+router.patch('/:meetupId/rsvps', Auth.verifyToken, validateId, tryCatch(rsvpController.respondToRsvp));
+router.delete('/:id', Auth.verifyToken, Auth.adminAuth, validateId, tryCatch(meetupController.deleteMeetup));
```
Comment (Veraclins) [inline server/routes/meetups.js]: Since `Auth.verifyToken` is applied to all the routes here, you can move it to the index file to remove the repetition.

**Review by Veraclins — CHANGES_REQUESTED** [review]
Great job! This is beginning to take a good shape. Keep it up. Just see the comments I left and attend to them and you should be good to go.

File: server/controller/comments.js (hunk 1 above)
Comment (oldmonad) [reply server/controller/comments.js]: I thought as much!, it's in the project rubric and I try to follow it as much as possible

**Pull request merged**